addons/payment_adyen/controllers/main.py

```diff
@@ -239,7 +239,7 @@ def adyen_notification(self):
         :return: The '[accepted]' string to acknowledge the notification
         :rtype: str
         """
-        data = request.jsonrequest
+        data = request.dispatcher.jsonrequest
```

Comment:
Do you confirm that we'll now be able to flag this route's type as "true JSON" and read the data from the kwargs?

Reply:
Also, there is the exact same kind of route in `payment_stripe.controllers.main.py`, only it uses `event = json.loads(request.httprequest.data)`.

Reply:
No, your statement is wrong. The various routes still are JSON-RPC and accessing `request.dispatcher.jsonrequest` is only a work-around. The response still is JSON-RPC serialized.

Reply:
Can't wait for the second refactor then :grimacing:

Reply:
cc @bouvyd